wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -133,29 +133,76 @@ BeatmapSetID (Integer) is the ID of the beatmap set.
 Difficulty
 ----------
 
-HPDrainRate (Decimal) specifies the HP Drain difficulty.
+The following 4 difficulty properties are expressed in *stars*, where 0 star means very easy, and 10 stars means very hard.
 
-`HPDrainRate:5`
+```
+HPDrainRate:4
+CircleSize:3.3
+OverallDifficulty:4
+ApproachRate:4.8
+```
+
+*HPDrainRate* specifies how fast the health bar decreases. At the time of this writing, the near future definition of this property remains unsettled.
```

Comment:
- put `HPDrainRate` in code instead of italics-> `` `HPDrainRate` `` (and for the others too)
- `At the time of this writing, the near future definition of this property remains unsettled.` -> `The near future definition of this property is unknown.` (or just remove the first part)

Reply:
Abusing `code` makes the document harder to read in my humble opinion.

When the article is complete, we could review its global shape for that kind of things. Right now I'm staying consistent with the other sections.

Reply:
the ASC says that keywords must be in code, but yeah, I guess it could wait since it is still WIP (I forgot that you're doing this in sections)